src/utils/verify-mantis-project.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs-extra';
+import Joi from 'joi';
+
+export const verifyMantisProject = () => {
```

Comment:
Could you describe what the `mantis.json` is to be used for? It seems we can get this information elsewhere.

`name`: This can be found in `package.json`
`version`: This can be found in `package.json` by inspecting the versions of the packages installed
`workspace`: What is this used for? If this is to give a relative directory to the root of the mantis project, why not expect all commands to be run within the nx workspace i.e. the root?
`description`: This can be found in `package.json`
`packageManager`: This can be inferred using `nypm`